tox.ini

```diff
@@ -14,6 +14,8 @@ skipsdist = True
 sitepackages = False
 # NOTE: Avoid false positives by not skipping missing interpreters.
 skip_missing_interpreters = False
+requires = pip < 20.3
```

Comment:
Re: the disclaimer at the top of the file. Is it now aligned with `release-tools` or are you diverging from it?